calculator.js

```diff
@@ -1,16 +1,19 @@
 function addNumberToDisplay(number) {
-	var displayScreen = document.getElementById('displayScreen');
-	var displayScreenValue = displayScreen.value
-	displayScreen.value = displayScreenValue + number
+	const displayScreen = document.getElementById('displayScreen');
```

Comment:
@denispop9 what was the reason behind changing from var to const out of interest?